substrate/frame/revive/rpc/src/client.rs

```diff
@@ -455,14 +455,14 @@ impl Client {
 	pub async fn submit(
 		&self,
 		call: subxt::tx::DefaultPayload<EthTransact>,
-	) -> Result<H256, ClientError> {
+	) -> Result<(), ClientError> {
```

Comment:
Not sure why this has been changed. This is from the first commit in this PR made by @athei.

Reply:
this is fine we don't really need to pass the substrate hash

Reply:
no, we don't use the substrate tx hash at the moment. We compute the ethereum tx hash and return that in anvil.

however, this could be useful to somebody later. Why do we break this API?